### DISCUSSION TITLE
通过本地Docker 环境安装higress，如何通过kubectl进行相关配置？

### DISCUSSION THREAD

**Discussion by feeops** [alibaba/higress#1610] [General]
通过本地Docker 环境安装higress，如何通过kubectl进行相关配置？

**Comment by CH3CHO** [comment]
既然没有走 K8s 部署，就没必要用 kubectl 来管理配置了。根据你的安装方式不同，可以在 Nacos 或者本地目录里直接管理相关配置。

**Reply by feeops** [reply]
用 kubectl本身会有一些校验，直接用文件不太安全吧？

**Reply by CH3CHO** [reply]
kubectl的校验也是API Server服务端做的，不过理论上非要用kubectl也是可以的，只需要把apiserver的对应端口暴露出来就行了。
